scipy/sparse/_csc.py

```diff
@@ -76,6 +76,9 @@ class csc_array(_cs_matrix):
       - slow row slicing operations (consider CSR)
       - changes to the sparsity structure are expensive (consider LIL or DOK)
 
+    Canonical format
+      - In the canonical format, CSC matrices have indices sorted per column,
+        and cannot contain duplicate entries.
```

Comment:
```suggestion
      - Indices are sorted by column.
      - There are no duplicate entries.
```

Reply:
I don't think this is quite right. This should be sorted in canonical:

``python
csc.indices[csc.indptr[i]:csc.indptr[1+1]]
```

So the row indices are sorted, but within each column

Reply:
How about:


```suggestion
      - Within each column indices are sorted by row
```

Reply:
LGTM